policy/modules/system/sysnetwork.te

```diff
@@ -77,6 +77,10 @@ manage_files_pattern(dhcpc_t, dhcpc_runtime_t, dhcpc_runtime_t)
 create_dirs_pattern(dhcpc_t, dhcpc_runtime_t, dhcpc_runtime_t)
 files_runtime_filetrans(dhcpc_t, dhcpc_runtime_t, { file dir })
 
+# create socket file for dhcpcd
+allow dhcpc_t self:unix_dgram_socket sendto;
```

Comment:
This should go up on line 65. Also increase it to `{ create_socket_perms sendto }`. The other perms are also covered by `logging_send_syslog_msg()` below.